vars/helm.groovy

```diff
@@ -1,12 +1,41 @@
+def getExtraCommands(chartName, tag) {
+  return "--set labels.version=$tag  --install --atomic"
```

Comment:
There is an extra space after `...$tag` and before `--install`

Reply:
Good spot, removed!